vulnerabilities/models.py

```diff
@@ -327,6 +327,14 @@ def get_or_create_from_purl(self, purl: PackageURL):
         package, _ = Package.objects.get_or_create(**purl_fields)
         return package
 
+    def get_from_purl(self, purl: PackageURL):
+        """
+        Return an existing or new Package (created if neeed) given a
```

Comment:
"(created if neeed) "?
Are you sure?

Reply:
You are right, we are not creating here.